jsonschema-validation.xml

```diff
@@ -491,6 +507,19 @@
                     </t>
                 </section>
 
+                <section title="requiredAllExcept">
+                    <t>
+                        The value of this keyword MUST be an array.
+                        Elements of this array, if any, MUST be strings, and MUST be unique.
```

Comment:
Why not require a minimum of 1, like `required`?

Reply:
An empty array is useful because it means all properties are required. If we have a minimum of 1, then we couldn't express that case.

Reply:
required` hasn't required an item since draft4.